packages/block-editor/src/components/use-resize-canvas/index.js

```diff
@@ -60,7 +60,7 @@ export default function useResizeCanvas( deviceType ) {
 					marginLeft: marginHorizontal,
 					marginRight: marginHorizontal,
 					height,
-					overflowY: 'auto',
```

Comment:
This dates back to #19082 which introduced device previewing and it seems at the time the canvas was not iframed. Now device previews are always iframed and it’s document itself is the scrolling context so this rule has no purpose.